File: python/examples/example_distance.py
from vesselfinder_api import VesselFinderApi
from vesselfinder_api.exceptions import ApiErrorException

v = VesselFinderApi(userkey='-- Input your userkey here --', errormode=False, save_last_info=True)

try:
    print(v.distance(_from="1.24703,51.94967", _to="28.68018,40.96205"))
except ApiErrorException as e:
    print(e)

File: php/examples/example_distance.php
<?php
include __DIR__ . '/../VesselFinderApi.php';

use API\VesselFinderApi;

$client = new VesselFinderApi('-- Input your userkey here --');

try {
    $distance = $client->distance("1.24703,51.94967", "28.68018,40.96205");
    print_r($distance);
} catch (\Exceptions\ApiErrorException $e) {
    print_r($e->getMessage() . PHP_EOL);
}